### PR TITLE
Add to docs what distinguishes Racetrack

### PR DESCRIPTION
**Pull request by iszulcdeepsense**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +17,33 @@ operationally sound and effective workloads without understanding the underlying
...
+- **Infrastructure independent**. Deploy to either a Kubernetes cluster
+  or a Docker environment in a single transparent step.
+- **Multiple job versions, immutable by design**. Immutable things tend to be more reliable.
+- **Customizable through plugins**,
+- **Reproducible jobs**. Racetrack ensures that anyone else can deploy the same job effortlessly.
+- **Out-of-the-box tools**: web endpoints, API documentation,
+  metrics, monitoring, tracing, and more.
+- **Production readiness**: authentication tokens, permissions management, scalability,
+- Released under a **permissive open-source license**,
+- Suitable for both **on-premises and cloud** environments.
```
[the remark this answers is not in the document]
Comment (iszulcdeepsense) [reply README.md:45]: Are you sure? I'm not an English expert, but here's the [article](https://andrewschoen.medium.com/software-people-of-the-world-stop-saying-on-premise-you-sound-dumb-56692dae871d) that says the opposite

> Premises: a house or building, together with its land and outbuildings (properties).
> Premise: an idea, theory, or theme that forms the basis for a statement or action.

File: README.md [hunk 2]
```diff
@@ -18,15 +17,33 @@ operationally sound and effective workloads without understanding the underlying
...
+What distinguishes Racetrack is:
+
+- **Server-side building**. Code is transformed into a microservice without your computer being involved.
+- You only supply your **function's logic**. No need to write repetitive API code, setting up 
+  webservers, creating dockerfiles, kubernetes YAMLs, and more. Racetrack takes care of that for you.
+- **Language agnostic**. Deploy code written in Python 3, Go, Rust,
+  or anything else encapsulated in a Dockerfile.
+- **Infrastructure independent**. Deploy to either a Kubernetes cluster
+  or a Docker environment in a single transparent step.
+- **Multiple job versions, immutable by design**. Immutable things tend to be more reliable.
```
[the remark this answers is not in the document]
Comment (iszulcdeepsense) [reply README.md:38]: Yes, in general, I meant that jobs are immutable and if you want to update it, you have to create a new one instead, which will be a part of the same family. This sentence is a bit clumsy, we can shorten it to:
"Immutable jobs"

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (LookACastle) [reply README.md:45]: I will take the loss on this one. https://en.wikipedia.org/wiki/On-premises_software even has the scathing sentence: On-premises software (abbreviated to on-prem, **and incorrectly referred to as on-premise**)

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (LookACastle) [reply README.md:38]: I like that a lot more, sounds cleaner.

File: README.md [hunk 3]
```diff
@@ -18,15 +17,33 @@ operationally sound and effective workloads without understanding the underlying
...
+- **Customizable through plugins**,
+- **Reproducible jobs**. Racetrack ensures that anyone else can deploy the same job effortlessly.
+- **Out-of-the-box tools**: web endpoints, API documentation,
+  metrics, monitoring, tracing, and more.
+- **Production readiness**: authentication tokens, permissions management, scalability,
```
[the remark this answers is not in the document]
Comment (iszulcdeepsense) [reply README.md]: Now I think it's risky to brag about being production ready, while it has never been seriously tested against security concerns (when exposed to public Internet), so I'll remove that line.

File: Makefile [hunk 4]
```diff
@@ -217,6 +217,9 @@ docker-clean-job:
 registry:
 	./utils/setup-registry.sh
 
+registry-down:
+	docker rm -f kind-registry || true
+
```
Comment (JosefAssadERST) [inline Makefile:222]: So Racetrack is a system for bringing down registries? Neat!

File: README.md [hunk 5]
```diff
@@ -18,15 +17,32 @@ operationally sound and effective workloads without understanding the underlying
...
+  or anything else encapsulated in a Dockerfile.
+- **Infrastructure independent**. Deploy to either a Kubernetes cluster
+  or a Docker environment in a single transparent step.
+- **Customizable through plugins**,
+- **Immutable jobs**
+- **Reproducible jobs**. Racetrack ensures that anyone else can deploy the same job effortlessly.
+- **Out-of-the-box tools**: web endpoints, API documentation,
+  metrics, monitoring, tracing, and more.
+- Released under a **permissive open-source license**,
+- Suitable for both **on-premises and cloud** environments.
```
Comment (JosefAssadERST) [inline README.md:44]: I'd be inclined to just choose 3 points even if they are all valid.

When the list is long it becomes its own job to even read the blurb. Most people will lose focus. It's not your fault, it's Twitter and Facebook and Youtube and TikTok.

While it's hard to choose, consider this: you're trying to make sure that there's something on that list for everyone, but that makes the list long and then you get no one.

My feeling is, numbers 2 3 and 4.

**Review by JosefAssadERST — APPROVED** [review]
I'm already approving, because I only want to give you an opinion, I don't demand that you agree.

File: Makefile (hunk 4 above)
Comment (iszulcdeepsense) [reply Makefile:222]: I realized that after doing all Quickstart steps, racetrack leaves this remnant registry, so I wanted to clean this up.

I'll rename this container to `racetrack-registry`, so people know it's associated with the racetrack's needs and there will be no risk of us destroying someone else's container.

Comment (JosefAssadERST) [reply Makefile:222]: I was actually being cheeky and I meant that this doesn't seem related to the issue this PR is aimed at.

File: README.md (hunk 5 above)
Comment (iszulcdeepsense) [reply README.md:44]: I agree, the list is quite long and 2,3,4 are the most catchy three. I'm going to move the rest to the User Guide doc (for more patient readers) cause I feel like this still explains important questions, but these questions will most likely arise at a later point.

**Pull request merged**